src/Npgsql/Internal/Converters/FullTextSearch/TsQueryConverter.cs

```diff
@@ -0,0 +1,219 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NpgsqlTypes;
+using static NpgsqlTypes.NpgsqlTsQuery.NodeKind;
+
+// ReSharper disable once CheckNamespace
```

Comment:
Can't say I love having these everywhere... Is there support for some resharper file in the converters directory alternatively?

Reply:
Not that I'm aware... Of course we can disable globally, but I don't think we want that either...